src/ci_providers/index.js

```diff
@@ -1,12 +1,14 @@
 const providerCircleci = require('./provider_circleci')
 const providerGitHubactions = require('./provider_githubactions')
 const providerLocal = require('./provider_local')
+const providerTravisci = require('./provider_travisci')
 
 // Please make sure provider_local is last
 const providers = [
   providerCircleci,
   providerGitHubactions,
-  providerLocal
+  providerLocal,
```

Comment:
Local has to go last. I know you want things in order, but it's the fallback if no prior ones match